firmware/setup.sh

```diff
@@ -1,5 +1,5 @@
 catkin build versavis
-shell=`grep ^$(id -un): /etc/passwd | cut -d : -f 7- |  cut -d'/' -f4-`
+shell=`echo $0 | cut -c 2-`
```

Comment:
small test script (test.sh) on my computer: 
```
shell=`echo $0 | cut -c 2-`                                                     
echo "setup.$shell"
```
then `./test.sh` on my computer prints: 
`setup./test.sh`

I think you rather wanted to have this:
```
shell=`echo $0 | rev | cut -c 1-2 | rev`                             
echo "setup.$shell"
```

which prints then 
`setup.sh`

But I'm not sure why you want take `$0`?